src/LinkLight.js

```diff
@@ -9,7 +9,6 @@ import styled from 'react-emotion'
 
 const LinkLight = styled(Link)`
   color: ${props => props.theme.colors.blue[3]} !important;
```

Comment:
Might not need `!important` now that you've refactored link to be a emotion.js component - I needed that to override the css previously.